.github/workflows/git-artifacts.yml

```diff
@@ -195,11 +232,21 @@ jobs:
           # Make sure that there is a `/usr/bin/git` that can be used by `makepkg-mingw`
           printf '#!/bin/sh\n\nexec /mingw64/bin/git.exe "$@"\n' >/usr/bin/git &&
 
-          # Restrict `PATH` to MSYS2 and to Visual Studio (to let `cv2pdb` find the relevant DLLs)
-          PATH="/mingw64/bin:/usr/bin:/c/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/:/C/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin${{matrix.arch.bin}}:/C/Windows/system32"
+          # We don't use `cv2pdb` on aarch64, which we build using Clang instead of GCC
+          if test "${{matrix.arch.name}}" != "aarch64"
+          then
+            # Restrict `PATH` to MSYS2 and to Visual Studio (to let `cv2pdb` find the relevant DLLs)
+            PATH="/mingw64/bin:/usr/bin:/c/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/:/C/Program Files (x86)/Microsoft Visual Studio 14.0/VC/bin${{matrix.arch.bin}}:/C/Windows/system32"
+            type -p mspdb140.dll || exit 1
+          else
+            # We don't want to build dashed built-ins anymore. Let's do this on aarch64 only to begin with
+            export SKIP_DASHED_BUILT_INS=YesPlease
```

Comment:
@dscho I thought this'd be a good opportunity to activate `SKIP_DASHED_BUILT_INS` which we discussed earlier. We could choose to enable it only on ARM64 for now to get some experience with it.

Reply:
I'd like to enable it in git-for-windows/git, regardless of the target architecture.

But because it is potentially disruptive a change, I would like to wait for a few days because I think we need a new Git for Windows release because of [a cURL problem that will supposedly be fixed this Wednesday](https://github.com/git-for-windows/git/issues/4171#issuecomment-1353502357) and because of [a Git Credential Manager problem that was also fixed in the meantime](https://github.com/GitCredentialManager/git-credential-manager/issues/993). There have been [some rumblings about an upstream maintenance release](https://lore.kernel.org/git/xmqqv8mewfqc.fsf@gitster.g/), too, so I definitely need to keep Git for Windows very stable for at least a few more days.

After that, I am more than happy to see `SKIP_DASHED_BUILT_INS = YabbaDabbaDoo` to the `MINGW` and the `Windows` parts of `config.mak.uname`.